### ISSUE TITLE
No autofocus when clicking "Reply in thread" if thread panel is already open

### ISSUE THREAD

**Issue by weeman1337** [vector-im/element-web#19821]
### Steps to reproduce

1. Use the latest dev version from here https://develop.element.io/
2. Enable Threads in Labs settings
3. Choose any message and click "Reply in thread"
4. Threads sidebar opens and the input field is focused
5. Click "Reply in thread" for any other message

#### What did you expect?

- Everytime I cllick "Reply in thread" the input field should be focused so that I can start typing

#### What happened instead?
 
- If the threads panel is already open, the input field doesn't receive the focus

### Operating system

Ubuntu 20.04.3 LTS

### Browser information

Firefox 94.0 (64-bit)

### URL for webapp

https://develop.element.io/

### Application version

Element version: b2e8f212e410-react-7f6f98443865-js-b2d83c1f80b5 Olm version: 3.2.3

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by SimonBrandner** [comment]
Possibly regressed by https://github.com/matrix-org/matrix-react-sdk/pull/7167
